app/controllers/support_interface/references_controller.rb

```diff
@@ -15,7 +15,7 @@ def confirm_cancel
     def reinstate; end
 
     def confirm_reinstate
-      RequestReference.new.call(@reference)
+      @reference.update!(feedback_status: 'feedback_requested', audit_comment: 'Reversing reference refusal using Support interface')
```

Comment:
we also need to set the `feedback_requested_at` timestamp to `nil` here

Reply:
Why's that? (Just curious)

Reply:
So we added some timestamp columns for data reporting purposes that track change in state. Previously we were having to dig through audits to determine how long references took to be provided/refused etc.

Reply:
OK. Why set it to `nil`, though — isn't that just discarding information?

Reply:
The idea of this is to revert the state change. I wouldn't have thought we would want a reference in the feedback_provided state with a feedback_refused_at timestamp?

I mean it doesn't hurt i guess, but if someone does something like

`ApplicationReference.where.not(feedback_refused_at: nil)` for reporting it's going to get picked up (although using the `ApplicationReferece.feedback_refused` enum scope would make more sense tbh)

Reply:
I like the idea of keeping as much info as possible - the fact that the timestamp was populated at some point is relevant info about the history of a given reference.

Fortunately the audit history will reflect that, so I think we're good to nullify the timestamp on the record itself.

Reply:
Yea i certainly think we should be retaining the audit 👍